pkgs/development/tools/misc/terramate/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+}:
+
+buildGoModule rec {
+  pname = "terramate";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "terramate-io";
+    repo = "terramate";
+    rev = "v${version}";
+    hash = "sha256-/e3NbUhHQlREjb3YYSfrPq/GE+BfFI3mUO4eRE4mk9U=";
+  };
+
+  vendorHash = "sha256-bja4v48BSPZC/vJVps2LKq90fReYpMY/yKTThsQQCHE=";
+
+  nativeBuildInputs = [ git ];
```

Comment:
What's git for?

Reply:
Something related to version. Removing it requires extra patching.